js-exercises/animal-sort/animalSort.js

```diff
@@ -0,0 +1,15 @@
+function animalSort(arr) {
+  const sortedByLegs = arr.sort(function (animalA, animalB) {
+    if (animalA.numberOfLegs !== animalB.numberOfLegs) {
+      return animalA.numberOfLegs - animalB.numberOfLegs;
+    } else if (animalA.numberOfLegs === animalB.numberOfLegs) {
```

Comment:
`else` would have worked instead of `else if`

Reply:
it's because i need to check one more condition else would not be sufficient for that, this is why i used `else if`